pkg/services/ssosettings/ssosettingsimpl/service.go

```diff
@@ -216,7 +216,7 @@ func (s *SSOSettingsService) getFallBackstrategyFor(provider string) (ssosetting
 func (s *SSOSettingsService) encryptSecrets(ctx context.Context, settings map[string]any) (map[string]any, error) {
 	result := make(map[string]any)
 	for k, v := range settings {
-		if isSecret(k) {
+		if api.IsSecret(k) {
```

Comment:
I think we can keep the `isSecret` function in the service file and move the section for hiding the secrets from the api to the service impl. We also need the service to return the full settings with secrets included so one suggestion would be to create a separate service func (something like `GetForProviderWithRedactedSecrets`).

Reply:
That makes sense, I'll move it into a separate service func